src/pages/petCare/components/care-body/careBody.interface.ts

```diff
@@ -0,0 +1,72 @@
+export interface CareBodyPet {
+  breed?: string;
+  weight?: string;
+  additonal?: string;
+  imgUrl?: [] | undefined;
```

Comment:
Why did you use `[] | undefined` instead of `string[]`?

Reply:
already fixed it.